planning/button/index.ftd

```diff
@@ -0,0 +1,90 @@
+-- my-ds.page: Button using `fastn` 🚧
+
+Video Title: How to create a button using `fastn`
+
+Owner: Ajit
+
+Audience: Frontend developer, designer
+
+Goal: To help users to create button using `fastn`
+
+Assumption: Have already installed `fastn` and create a fastn package.
+Understanding of datatypes, components.
+
+
+-- ds.h1: Part 1 - Create Button
+
+In this part we will create simple UI for button.
+
+-- ds.h2: Intro Clip
+
+Screen: Introduction slide
+
+Hey Guys, Welcome to the video where we are going to create a button using
+`fastn`. We will do it by creating a component and at first we are going to use
+only the required properties to it. 
+
+
+-- ds.h2: button.ftd file
+
+Screen
+
+Button UI
+
+-- ds.image:
+src: $fastn-assets.files.planning.button.button-shadow.png
+
+Script
+
+We have taken inspiration to create a UI similar to what you can see. I found
+this UI of button in Vercel. In this video we will create a basic button that
+displays just the text and give a url to these buttons.
+
+In the next video, we will polish the UI and try to re-create similar UI.
```

Comment:
Make all videos standalone. You do not know what is the next video, referring to video number or what comes after or earlier makes it hard to reuse a video.

Reply:
@gargajit if you have done the changes asked for, click on the resolve button.

Reply:
@amitu I understand. Why I did not resolve these comments is because I have altered the script which will be covered in one video.
Resolving now.